pages/mypage/index.module.scss

```
@@ -0,0 +1,66 @@
+@use '@/styles/index' as *;
+
+.text {
+  display: flex;
+  flex-direction: column;
+  padding-left: 12px;
+  padding-right: 12px;
+  padding-top: 16px;
+  padding-bottom: 16px;
```

```suggestion
padding: 16px 12px;
```
padding 4가지 방향을 다 주실거면 불필요하게 나눈 부분같아요~!